src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/caseworker/event/page/ManageHearings.java

```diff
@@ -23,6 +24,8 @@
  * Contains method to add Page Configuration for ExUI.
  * Display the Manage hearings Details screen with all required fields.
  */
+@Slf4j
+@Component
```

Comment:
Do we need this annotation here? as we are not using any object from spring context here